addons/html_editor/static/src/core/delete_plugin.js

```diff
@@ -600,32 +600,29 @@ export class DeletePlugin extends Plugin {
     // If joined, the range is collapsed to start.
     // Returns the updated range.
     joinFragments(range) {
-        const { joinableLeft, joinableRight } = this.getJoinableFragments(range);
         const { commonAncestorContainer: commonAncestor } = range;
+        const { joinableLeft, joinableRight } = this.getJoinableFragments(range);
+        const [left, right] = [joinableLeft.node, joinableRight.node];
+        const scenario = joinableLeft.type + " + " + joinableRight.type;
```

Comment:
not sure it's really necessary to create that string and use a switch case instead of normal if/else, but I don't have a strong opinion here

Reply:
I know, it was just for improved readability. I might have exagerated...